markets/perps-market/contracts/storage/GlobalPerpsMarketConfiguration.sol

```diff
@@ -207,7 +207,7 @@ library GlobalPerpsMarketConfiguration {
         address referrer,
         PerpsMarketFactory.Data storage factory
     ) private returns (uint256 referralFeesSent) {
-        if (referrer == address(0)) {
+        if (fees == 0 || referrer == address(0)) {
```

Comment:
nit: feel like this check should exist at the caller level.  in `collectFees()` we should return (0,0) if the orderFees == 0.  this function shouldn't even be called if there's no fees to collect.